libcaf_core/caf/uri.cpp

```diff
@@ -135,6 +135,11 @@ class nop_builder {
     return *this;
   }
 
+  template <class T, class P>
+  nop_builder& userinfo(T&&, P&&) {
```

Comment:
Can have one or two arguments, no?

```suggestion
  template <class... Ts>
  nop_builder& userinfo(Ts&&...) {
```